pkg/kubeclient/client.go

```diff
@@ -1,39 +1,67 @@
 package kubeclient
 
 import (
-	"k8s.io/client-go/kubernetes"
+	crtapi "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/client-go/rest"
 )
 
-type KubeClient struct {
-	CoreClient kubernetes.Interface
-}
+func NewCRTV1Alpha1Client(cfg *rest.Config, namespace string) (*CRTV1Alpha1Client, error) {
```

Comment:
could we add some documentation to our public apis?

Reply:
Done